lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
+    end
+
+    #Supports carrying over DSL attributes to the instantiated
+    #parent class
+    def initialize
```

Comment:
initialize logic added by modules seems dangerous. 

I can think of few libraries that do it but it's always been weird to track it down.

It's just my gut, I'm willing to be wrong